src/django/api/models/facility/facility.py

```diff
@@ -293,3 +294,9 @@ def current_tile_cache_key():
             tile_version = 0
 
         return f'{timestamp}-{tile_version}'
+
+    @staticmethod
+    def update_facility_updated_at_field(facility_id):
+        facility = Facility.objects.get(pk=facility_id)
+        facility.updated_at = timezone.now()
+        facility.save()
```

Comment:
@roman-stolar , @vladsha-dev 
I'm not sure, but shouldn't `updated_at` field be updated automatically on `facility.save()`? I mean, `facility.updated_at = timezone.now()` looks redundant.

Reply:
In src/django/api/models/facility/facility.py
`updated_at = models.DateTimeField(auto_now=True, db_index=True)`
`auto_now=True` means that this field should be updated automatically on `facility.save()`

Reply:
Rewritten as:
    def update_facility_updated_at_field(facility_id):
        facility = Facility.objects.get(pk=facility_id)
        facility.save(update_fields=['updated_at']) - update only one field to avoid unnecessary load on the database

Reply:
Yes, this looks more correct.